docs/install/index.md

```diff
@@ -39,10 +39,11 @@
 Indicate your preferred configuration. Then, follow the customized commands to install MXNet.
 
 <div class="dropdown">
-  <button class="btn current-version btn-primary dropdown-toggle" type="button" data-toggle="dropdown">v1.4.0
+  <button class="btn current-version btn-primary dropdown-toggle" type="button" data-toggle="dropdown">v1.4.1
   <span class="caret"></span></button>
   <ul class="dropdown-menu opt-group">
-    <li class="opt active versions"><a href="#">v1.4.0</a></li>
+    <li class="opt active versions"><a href="#">v1.4.1</a></li>
+    <li class="opt versions"><a href="#">v1.4.0</a></li>
```

Comment:
Do we still need to provide option for 1.4.0? Previously we kept option for the last patch release of a minor release.

Reply:
Good catch! I was not aware of it before. @szha Could you advise?

Reply:
Let's stick to the convention and provide the latest patch release of each minor release.